integration/sawtooth_integration/tests/test_basic_auth_proxy.py

```diff
@@ -68,16 +68,17 @@ def _assert_valid_authed_request(self, url):
         try:
             response = urlopen(request, context=context)
         except HTTPError as e:
+            LOGGER.error('An error occured while requesting {}'.format(url))
```

Comment:
When using `LOGGER.error` you should use the printf-style formatting:
`LOGGER.error('An error occurred while requesting %s', url)`
At least pylint always seems to complain when I don't ;)